### DISCUSSION TITLE
Thanks for this!

### DISCUSSION THREAD

**Discussion by freiguy1** [godot-zig/godot-zig#13] [General]
Hello! I just got the latest code from main branch, did `zig build`, opened the project in Godot 4.2, clicked run, and it... just... worked. I kept Godot open, modified the # of sprites bouncing to 10, ran `zig build`, re-started in Godot 4.2, and sure enough the sprite # was lowered. Also it was surprising to see suggestions/auto-complete working with zls for Godot internals:
![image](https://github.com/godot-zig/godot-zig/assets/613651/da72ebc6-aaa3-4bc7-963f-943fc7c39908)

This seems like it could be the marriage of two really cool open source tools to make high performance games.

That's all this is, a thank you note :)

**Comment by elvisxzhou** [comment]
Glad you like it, as zig matures, tooling in zig ecosystem, including godot-zig,  should become increasingly stable and usable.

**Comment by scripturial** [comment]
I just wanted to second this. Thanks so much for everyone has put work in this so far. Im excited to learn zig and try and see how I can get it working in Godot.  Sorry in advance for any noob questions. :)
